src/test/resources/application-test.properties

```diff
@@ -0,0 +1,4 @@
+#--------------------- DB Connection ------------------
```

Comment:
Since this file will never change, can you rename it to `application.properties`?

This folder structure will be used only for testing

Reply:
Done.